docker/Transformation-Engine/src/main/kotlin/net/nhsd/fhir/converter/transformer/careconnect_transformers.kt

```diff
@@ -91,7 +94,18 @@ internal val careconnectTransformers: HashMap<String, ExtensionTransformer> = ha
     CARECONNECT_EVIDENCE_URL to ::evidence,
     CARECONNECT_ALLERGY_ASSOCIATED_ENCOUNTER_URL to ::associatedEncounter,
     CARECONNECT_ALLERGY_INTOLERANCE_END_URL to ::allergyIntoleranceEnd,
+    CARECONNECT_GPC_ALLERGY_INTOLERANCE_END_URL to :: allergyIntoleranceEndGPC
```

Comment:
Minor whitespace nitpick here, the other map entries don't have a gap between `::` and the function to be invoked.